.github/workflows/release.yaml

```diff
@@ -81,3 +88,32 @@ jobs:
               env:
                 GITHUB_TOKEN: ${{ secrets.CODEFLARE_MACHINE_ACCOUNT_TOKEN }}
               shell: bash
+
+            - name: Append tag to Snyk monitoring list
+              run: |
+                sed -i 's/list_of_released_tags=(/list_of_released_tags=("v${{ github.event.inputs.release-version }}", /' .github/workflows/snyk-security.yaml
+
+            - name: Commit and push changes
+              run: |
+                git config --global user.email "138894154+codeflare-machine-account@users.noreply.github.com"
+                git config --global user.name "codeflare-machine-account"
+                git checkout -b $PR_BRANCH_NAME
+                git commit -am "Update snyk-security.yaml"
+                git push --set-upstream origin "$PR_BRANCH_NAME"
+
+            - name: Create Pull Request
+              run: |
+                  gh pr create \
+                  --title "$pr_title" \
+                  --body "$pr_body" \
+                  --head ${{ env.PR_BRANCH_NAME }} \
+                  --base main \
+                  --label "lgtm" \
+                  --label "approved"
```

Comment:
**This PR consists of two items:**
1. The `snyk-security.yaml`: Uploads snapshots of supported release tags to Snyk UI for monitoring.
2. Changes in `release.yaml`: The `snyk-security.yaml` contains a list of release tags that we continue to support. The PR created here is to automatically add to the list any new release tags to avoid the need of doing this manually.

**Current Behaviour:**
The `snyk-security.yaml` currently runs on `push` events. This takes a snapshot of the dependencies for each tag we support, then uploads this to the Snyk UI, where then Snyk continues to monitor each release tag that we uploaded. The importance of running this on push events is mainly for the `main` branch, to update the snapshot and re-upload the latest dependencies to Snyk UI for monitoring. However, in its current state it will essentially re-take snapshots of all the tags in the list of supported tags.

I have a few ideas and potential enhancements: We could re-take a snapshot of only the `main` branch, on push events to `main`. The other scenario to this would be to take snapshots of new release tags. We would need to perform an initial (once-off) snapshots of all tags we support and upload them to Snyk UI, then we let the release action take the snapshot of any new release tags for monitoring. 

**Some thoughts:**
Based on the potential enhancements, the reason we could take snapshots on release is because RHOAI will always use the released version of the `codeflare-sdk`. With Snyk we continue to monitor those tags for any future CVEs that may arise - We would only address Critical CVEs as RHOAI only performs `z` releases on Critical CVEs.

Dependabot is still required to automatically create PRs for the `main` branch and be up-to-date proactively.

Let me know what you think on the potential enhancements, happy to further discuss on this on a call, thanks Varsha :)